src/tools/rest-frontend/public/assets/translations/en.json

```diff
@@ -0,0 +1,370 @@
+    "APP.AUTH.LOGIN.LABEL.VALIDATION.USERNAME.MAXLENGTH": "Your entered username is too long and cannot be processed.",
+    "APP.AUTH.LOGIN.LABEL.VALIDATION.USERNAME.PATTERN": "Your entered username contains unsupported characters.",
+    "APP.AUTH.LOGIN.LABEL.VALIDATION.USERNAME.REQUIRED": "Please enter your username to login.",
+    "APP.AUTH.LOGIN.NOTIFICATION.HEADER": "Login",
+    "APP.AUTH.LOGIN.NOTIFICATION.MESSAGE.AUTH_INVALID_PASSWORD": "The entered password is wrong, please correct your input.",
+    "APP.AUTH.LOGIN.NOTIFICATION.MESSAGE.AUTH_UNKNOWN_USERNAME": "There is no account registered for the entered username.",
+    "APP.AUTH.LOGIN.NOTIFICATION.MESSAGE.SUCCESS": "Login successful!",
+    "APP.AUTH.LOGIN.PANEL.HEADING": "Authentication",
+    "APP.AUTH.LOGIN.PLACEHOLDER.USERNAME": "Enter your username",
+    "APP.AUTH.LOGIN.PLACEHOLDER.PASSWORD": "Enter your password",
+    
+    "APP.AUTH.LOGOUT.NOTIFICATION.HEADER": "Logout",
+    "APP.AUTH.LOGOUT.NOTIFICATION.MESSAGE.SUCCESS": "Logout successful!",
+    
+    "APP.AUTH.REGISTRATION.BUTTON.LOGIN": "Go to login",
+    "APP.AUTH.REGISTRATION.BUTTON.SUBMIT": "Register account",
+    "APP.AUTH.REGISTRATION.LABEL.EMAIL": "Email:",
+    "APP.AUTH.REGISTRATION.LABEL.PASSWORD": "Password:",
+    "APP.AUTH.REGISTRATION.LABEL.PASSWORD_CONFIRMATION": "Confirm password:",
+    "APP.AUTH.REGISTRATION.LABEL.USERNAME": "Username:",
+    "APP.AUTH.REGISTRATION.LABEL.VALIDATION.EMAIL.EMAIL": "You have to enter a valid email address.",
+    "APP.AUTH.REGISTRATION.LABEL.VALIDATION.EMAIL.REQUIRED": "You have to enter an email address.",
+    "APP.AUTH.REGISTRATION.LABEL.VALIDATION.PASSWORD.MINLENGTH": "The password has to be at least eight characters long.",
+    "APP.AUTH.REGISTRATION.LABEL.VALIDATION.PASSWORD.PATTERN": "The password must contain at least one lower-case letter, one upper-case letter and one digit. We recommend also using special characters as well.",
+    "APP.AUTH.REGISTRATION.LABEL.VALIDATION.PASSWORD.REQUIRED": "You have to enter a password.",
+    "APP.AUTH.REGISTRATION.LABEL.VALIDATION.PASSWORD_CONFIRMATION.EQUAL_INPUT": "Your confirmation does not match the password.",
+    "APP.AUTH.REGISTRATION.LABEL.VALIDATION.PASSWORD_CONFIRMATION.REQUIRED": "You have to confirm your password.",
+    "APP.AUTH.REGISTRATION.LABEL.VALIDATION.USERNAME.MAXLENGTH": "The username may not exceed 20 characters.",
+    "APP.AUTH.REGISTRATION.LABEL.VALIDATION.USERNAME.MINLENGTH": "The username has to be at least three characters long.",
+    "APP.AUTH.REGISTRATION.LABEL.VALIDATION.USERNAME.PATTERN": "The username may only contain letters, digits, dashes and dots.",
+    "APP.AUTH.REGISTRATION.LABEL.VALIDATION.USERNAME.REQUIRED": "You have to enter a username.",
+    "APP.AUTH.REGISTRATION.NOTIFICATION.HEADER": "Account Registration",
+    "APP.AUTH.REGISTRATION.NOTIFICATION.MESSAGE.REQUEST_MALFORMED_DATA": "An error occured. Please contact the administrator, if the error persists.",
```

Comment:
This error should never ever occur and the error message is only implemented for sake of completeness. The frontend will always send a json object that is properly formed (especially after form validation). The only situation I can think of where this might occur is when some data corrupts during transmissing... but yeah.